index.ts

```diff
@@ -1,6 +1,9 @@
+import { Config } from "./src/Config.js";
 import Server from "./src/Server.js";
 
-const server = new Server(25566);
+const config: Config = await Config.fromFile("config.json");
```

Comment:
here it would be best imo to `path.join(Server.path, config.json)`

Reply:
somehow this breaks compatibility with windows, it worked on arch but does not on windows, it appends a `\` at the start for some odd reason, will just realise it as it is rn